src/components/profession-hero/ProfessionHero.css

```diff
@@ -1,225 +1,222 @@
-
 .profession {
-    display: flex;
-    flex-direction: column;
-    padding: 95px 0 100px 0;
-    max-width: 1440px;
-    margin: 0 auto;
-    box-sizing: border-box;
+  display: flex;
```

Comment:
I'm not sure what happened here but it looks like an auto-formatter reformatted the spacing on every line. Any idea what might have caused this?

Reply:
I have no idea why, this is the first time this has happened... I'm gonna fix this now

Reply:
It actually happened since the last PR :(

Reply:
Ooh I see... When Colin built these components he used another format (4 spaces instead of 2), and apparently we never changed it. The PageNotFound.css has the same incorrect spacing, so I'll fix this and add these changes to my PR. All other pages looked fine 👍

Reply:
Ohhh ok that makes sense. Thanks Denise.